firebase-functions/src/main/java/com/google/firebase/functions/HttpsCallableReference.kt

```diff
@@ -0,0 +1,159 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.google.firebase.functions
+
+import com.google.android.gms.tasks.Task
+import java.net.URL
+import java.util.concurrent.TimeUnit
+
+/** A reference to a particular Callable HTTPS trigger in Cloud Functions.  */
+class HttpsCallableReference {
+  // The functions client to use for making calls.
+  private val functionsClient: FirebaseFunctions
+
+  // The name of the HTTPS endpoint this reference refers to.
+  // Is null if url is set.
+  private val name: String?
+
+  // The url of the HTTPS endpoint this reference refers to.
+  // Is null if name is set.
+  private val url: URL?
+
+  // Options for how to do the HTTPS call.
+  private val options: HttpsCallOptions
```

Comment:
this should actually be `package-private`, did you manually put the `private` here? Doesn't really matter in this case, but interesting thing to look out for.

Reply:
I didn't touch this privacy, the converter must've opted to turn a package variable to a private one

Reply:
This became visible for testing